.golangci.yml

```diff
@@ -0,0 +1,69 @@
+run:
+  build-tags: 
+    - aws
+    - packet
+    - aks
+    - e2e
+    - baremetal
+    - disruptivee2e
+    - poste2e
+
+  modules-download-mode: vendor
+
+  timeout: 5m
+
+issues:
+  exclude-use-default: false
+  new-from-rev: HEAD~
```

Comment:
This won't work, as it will only be checking last commit and we want to check everything on top of master branch.